exercises/clock/.meta/gen.go

```diff
@@ -44,6 +45,10 @@ type testGroup struct {
 	} `property:"equal"`
 }
 
+func (groups testGroup) GroupShortName() string {
+	return strings.ToLower(strings.Fields(groups.Description)[0])
```

Comment:
be careful here with a variable name dependent on description - for example if someone changes the description to be something like "the following tests will do something" then the variable name becomes `theTests` which doesn't look good. Perhaps we will defer on dealing with that when it happens.